foreshadow/utils/__init__.py

```diff
@@ -6,10 +6,12 @@
     get_config_path,
     get_transformer,
 )
+from foreshadow.utils.testing import dynamic_import
```

Comment:
Should we be exporting a testing import to the utils name_space? If this is needed, then shouldn't we move it to common

Reply:
#124